instana/propagators/base_propagator.py

```diff
@@ -130,15 +130,22 @@ def _get_participating_trace_context(self, span_context):
         traceparent = span_context.traceparent
         tracestate = span_context.tracestate
         traceparent = self._tp.update_traceparent(traceparent, tp_trace_id, span_context.span_id, span_context.level)
+
+        # In suppression mode do not update the tracestate and
+        # do not add the 'in=' key-value pair to the incoming tracestate
+        # Just propagate the incoming tracestate (if any) unchanged.
+        if span_context.level == 0:
```

Comment:
Since we have a helper method introduced now
```suggestion
        if span_context.suppression:
```